api/src/components/link/service.ts

```diff
@@ -8,6 +9,20 @@ export const create = async (fromPublicationId: string, toPublicationId: string)
         }
     });
 
+    const latestVersionFrom = await client.prisma.publicationVersion.findFirst({
```

Comment:
We already have a service for this:
`const latestVersionFrom = await publicationVersionService.get(fromPublicationId, 'latest');`